src/internal/dnd-controller/controller.ts

```diff
@@ -63,36 +64,47 @@ interface Transition {
 class DragAndDropController extends EventEmitter<DragAndDropEvents> {
   private droppables = new Map<ItemId, Droppable>();
   private transition: null | Transition = null;
+  private sourceId = "";
```

Comment:
Can we add sourceId to the transition object instead? To make sure those never get out of sync.